packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -358,6 +357,11 @@ class ThemeData with Diagnosticable {
       'This feature was deprecated after v3.21.0-10.0.pre.',
     )
     ButtonBarThemeData? buttonBarTheme,
+    @Deprecated(
+      'Use [DialogThemeData.backgroundColor] instead. '
```

Comment:
I don't think the brackets will create breadcrumbs like in dartdocs

```suggestion
      'Use DialogThemeData.backgroundColor instead. '
```

Reply:
Good catch!